### DISCUSSION TITLE
New accelerated sampling algorithms

### DISCUSSION THREAD

**Discussion by lnnbig** [deepmodeling/dpgen#1046] [Ideas]
Hello developers,

Active learning methods progress rapidly and other groups have actively contributed to developing new accelerated sampling methods including two recent works using uncertainty biasing: (1) https://arxiv.org/abs/2210.04225, and (2) https://www.researchsquare.com/article/rs-2109927/v1. 

While DP-GEN gains significant popularity, hope to see this code becomes more successful with diverse sampling methods to be incorporated in future updates.

**Comment by wanghan-iapcm** [comment]
Thank you very much for the suggestion and the references. You are definitely welcome if you want to contribute
